lib/FocusTrap.js

```diff
@@ -11,22 +13,52 @@ const FocusTrap = React.createClass({
 
   getDefaultProps() {
     return {
-      component: 'div'
+      component: DOCUMENT_FRAGMENT
     };
   },
 
+  onFocus(...args) {
+    const {children, onFocus} = this.props;
+
+    const child = React.Children.only(children);
+    if (child.onFocus) {
+      child.onFocus(...args);
+    }
+    if (onFocus) {
+      onFocus(...args);
+    }
+  },
+
+  onBlur(...args) {
+    const {children, onBlur} = this.props;
+
+    const child = React.Children.only(children);
+    if (child.onBlur) {
+      child.onBlur(...args);
```

Comment:
Calling methods on children is a bit anti-react as it breaks the direction of data.